package.json

```diff
@@ -31,10 +33,12 @@
     "@types/react": "17.0.36",
     "@typescript-eslint/eslint-plugin": "4.33.0",
     "babel-jest": "27.3.1",
+    "cypress": "9.1.0",
     "eslint": "7.32.0",
     "eslint-config-next": "12.0.4",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-prettier": "4.0.0",
+    "har-validator": "5.1.5",
```

Comment:
Workaround for this issue: https://github.com/cypress-io/cypress/issues/19102
Otherwise the `cypress` package could not be installed.